modules/ROOT/pages/permissions-by-product.adoc

```diff
@@ -0,0 +1,135 @@
+= Permissions Available in Anypoint Platform
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following permissions can be assigned to xref:teams.adoc[teams], granted to xref:users.adoc#grant-user-permissions[individual users], or combined to xref:roles.adoc#create-custom-roles[create custom roles].
+
+== General Permissions
```

Comment:
@cmordue if/when we rename this section to Access Management let's remember to rename it here, too